planning/behavior_velocity_planner/config/occlusion_spot.param.yaml

```diff
@@ -9,7 +9,8 @@
         lateral_distance: 1.5               # [m] maximum lateral distance to consider hidden collision
       motion:
         safety_ratio: 0.8                   # [-] jerk/acceleration ratio for safety
-        max_slow_down_acceleration: -1.5    # [m/s^2] minimum deceleration for safe deceleration.
+        max_slow_down_jerk: -0.7            # [m/s^3] minimum jerk deceleration for safe brake.
+        max_slow_down_accel: -2.5           # [m/s^2] minimum accel deceleration for safe brake.
```

Comment:
WIth jerk limit acceleration slow down limit can be lower but max slow down accel can also be -2.0 or so